SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialSender.h

```diff
@@ -111,6 +111,8 @@ class SPATIALGDK_API USpatialSender : public UObject
 	void UpdateInterestComponent(AActor* Actor);
 
 	void ProcessOrQueueOutgoingRPC(const FUnrealObjectRef& InTargetObjectRef, SpatialGDK::RPCPayload&& InPayload);
+	// Attempt to send all outgoing RPCs.
+	void ProcessOutgoingRPCs();
```

Comment:
Do we still need to expose this function?

Reply:
Good point, we can remove this.